packages/calcite-components/src/components/combobox/combobox.e2e.ts

```diff
@@ -2002,4 +2002,22 @@ describe("calcite-combobox", () => {
     await combobox.press("Enter");
     expect(chips.length).toBe(2);
   });
+
+  it("prevents opening a readonly combobox", async () => {
+    const page = await newE2EPage({
+      html: html`
+        <calcite-combobox id="myCombobox" read-only="true">
+          <calcite-combobox-item value="Raising Arizona" text-label="Raising Arizona"></calcite-combobox-item>
+          <calcite-combobox-item value="Miller's Crossing" text-label="Miller's Crossing"></calcite-combobox-item>
+          <calcite-combobox-item value="The Hudsucker Proxy" text-label="The Hudsucker Proxy"></calcite-combobox-item>
+          <calcite-combobox-item value="Inside Llewyn Davis" text-label="Inside Llewyn Davis"></calcite-combobox-item>
+        </calcite-combobox>
+      `,
+    });
+
+    const combobox = await page.find("calcite-combobox");
+    expect(await combobox.getProperty("open")).toBeFalsy();
+    await combobox.click();
```

Comment:
nitpick: We might need to call `await page.waitForChanges();` after this call. Otherwise looks good.